File: JanperCab/3-Application/Dtos/Enquiry/EnquiryForPickUpSheetDto.cs
namespace _3_Application.Dtos.Enquiry
{
    public class EnquiryForPickUpSheetDto : EnquiryForSheetDto
    {
        public int? PickUpSheetId { get; set; }

        public EnquiryForPickUpSheetDto(_1_Domain.Enquiry enquiry) : base(enquiry)
        {
            PickUpSheetId = enquiry.PickUpSheetId;
        }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/enquiry/EnquiryForPickUpSheetDto.ts
import { EnquiryForSheetDto } from './EnquiryForSheetDto';

export class EnquiryForPickUpSheetDto extends EnquiryForSheetDto {
  pickUpSheetId: number;

  constructor() {
    super();
    this.$type =
      '_3_Application.Dtos.Enquiry.EnquiryForPickUpSheetDto, 3-Application';
  }
}
